docs/writing_planemo_galaxy_wrapper.rst

```diff
@@ -0,0 +1,222 @@
+*Galaxy wrapper generation with Planemo*
+
+
+**************************
+Prerequisites
+**************************
+Installation of 
+-  pip
+- virtualenv
+- planmeo
+
+Before installing planemo it is prerequisite that you have latest version of pip 7.0 or greater and virtualenv-1.9 or greater
+
+To install globally with pip (if you have pip 1.3 or greater installed globally):
+
+.. code-block:: bash
+
+[sudo] pip install virtualenv
+
+Refer to user documents for further information https://virtualenv.pypa.io/en/stable/installation/
+
+Setting up virtual environment Installation of planemo
+
+.. code-block:: bash
+
+$ virtualenv .venv; . .venv/bin/activate
+$ pip install "pip>=7" # Upgrade pip if needed.
+$ pip install planemo
+ 
+https://planemo.readthedocs.io/en/latest/readme.html#obtaining
+
+After installation you can test if planemo
+
+
+Installing bioconda packages
+
+Developing galaxy tools wrappers for a particular bioconda recipe requires installation of the package in the same environment.  
```

Comment:
I thought that planemo/galaxy did this for you directly (and that you don't need to install the conda package for the tool manually). Any comments @nh3 @cartal ? Thanks

Reply:
I think planemo/galaxy handles the installation of packages required to run the tools. However, to generate test-data for planemo to test against, one needs to be able to run the tools outside planemo/galaxy. For that, the required package is needed, though it can be in a separate environment from planemo. In my hand, I cannot get planemo to work properly within a conda environment (which is also labelled as experimental on its installation guide). I had to follow the pip route and install it using virtualenv, but it might just be me.